packages/plugin-ext/src/plugin/tests.ts

```diff
@@ -197,7 +198,7 @@ export class TestControllerImpl implements theia.TestController {
             .filter(isDefined);
 
         const request = new TestRunRequest(
-            includeTests, excludeTests, profile, false // don't support continuous run yet
+            includeTests, excludeTests, profile, false /* don't support continuous run yet */, preserveFocus
         );
 
         const run = this.testRunStarted(request, name, false, false);
```

Comment:
We are not notifying the UI with `$notifyOnTestRunRequest`. Why not?

Reply:
The `$notifyOnTestRunRequest` has been replaced by the `$notifyOnTestRunCreated`. It should now be always notified, regardless where the TestRun is started.